inspirehep/ext/redis/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
Sorry, it was not clear with our comments. We were discussing the license header. The file can just be empty.